LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 California Institute of Technology
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Is this now autofilled or?

Reply:
Oops. I can update with Caltech copyright since Cristina was the original creator of this dictionary